docs/introduction/understanding-docker.md

```diff
@@ -21,7 +21,7 @@ running code.
 Docker does this by combining a lightweight container virtualization platform
 with workflows and tooling that help you manage and deploy your applications.
 
-At its core, Docker provides a way to run almost any application securely
+At its core, Docker provides a way to run almost any applications securely
```

Comment:
Thank you for your opinion. 
If I still want to contribute to this document, should I pull a new request that only modify "AND" to "and", "be" to "is", ", and;" to "." ?